class AlreadyExistsError(Exception):
    pass


class EntityNotFoundError(Exception):
    pass
